src/Sentry/Internal/SerializableExtensions.cs

```diff
@@ -4,18 +4,18 @@
 using System.Threading.Tasks;
 using Sentry.Protocol.Envelopes;
 
-namespace Sentry.Tests.Helpers
+namespace Sentry.Internal
 {
     internal static class SerializableExtensions
```

Comment:
I may be wrong, but moving a static code from the Test project to the Main project isn't going to include an unused static extension to the Main SDK instead of limiting it to the Test projects.